src/components/Header/_header.scss

```diff
@@ -1,17 +1,17 @@
 @import '#scss/vars';
+@import '#scss/utils';
 
 .#{$px}-header {
   align-items: center;
   background-color: $white;
   border-bottom: 1px solid rgba(0, 0, 0, 10%);
   display: flex;
-  height: 60px;
-  padding: 0 $spacing-md;
-  position: fixed;
-  top: 30px;
-  width: 100vw;
+  padding: $user-management-height $spacing-md 0; // save space for the user management
+  position: sticky;
+  top: 0;
 
-  @media (max-width: #{$breakpoint-md - 1}) {
+  @include isHeaderMobile {
```

Comment:
I love this. So much more readable at a glance 👍

Reply:
yeah the gray background is there in the newer Figmas.  Great catch on the missing overlay, I had caused this with the wrong `isHeader*` mixin.  I've updated the screenshot at the top to match.

Reply:
here's the updated Figma: https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=7954-18333&t=JnDVzGnoZFwGuUW8-4. As far as the centering of the Browse All Upcoming, since that's content provided by the consumer, I couldn't think of a good way to make that change in seldon itself so only do it in Remix directly 🤷

Reply:
Ahh ok I see what you're saying. We make tweaks to it directly in remix. I think this is fine then